v2/brigadier/src/groups.ts

```diff
@@ -1,45 +1,93 @@
-import { Job } from "./jobs"
+import { Runnable } from "./runnables"
 
-export class Group {
+/**
+ * The base type for Runnables that contain multiple other Runnables.
+ * Do not construct the base Group type; use Job.sequential (or SerialGroup)
+ * or Job.concurrent (or ConcurrentGroup) instead.
```

Comment:
I think this is another case where you're going to know better than me what the best JS/TS approach is here, so consider this a question more than a comment...

`Group` doesn't get re-exported from `index.ts`. Unless I am mistaken (and I well could be), this makes direct instantiation of this `Group` base class by a user of the Brigadier package an impossibility. Or doesn't it?

If I'm right, we could lose the admonishment not to instantiate this directly. If I'm wrong, then forget I said anything.

And this is non-blocking anyway.

Reply:
I think you are right. So we could indeed lose the admonition. Though in turn this people probably won't see these docs, so it does no harm! So I'm currently thinking - leave the admonition in, not so much for end users but more for people who are _reading the source code_ and wondering about the role of the Group class. Is that potentially useful, or just absurdly defensive?

Reply:
It sounds fine to me.